app/src/main/java/at/aau/edu/wizards/ui/scoreboard/recycler/ScoreboardItemViewHolder.kt

```diff
@@ -0,0 +1,17 @@
+package at.aau.edu.wizards.ui.scoreboard.recycler
+
+import androidx.recyclerview.widget.RecyclerView
+import at.aau.edu.wizards.R
+import at.aau.edu.wizards.databinding.ItemScoreboardBinding
+import at.aau.edu.wizards.ui.scoreboard.Scoreboard
+
+class ScoreboardItemViewHolder(
+    private val binding: ItemScoreboardBinding
+) : RecyclerView.ViewHolder(binding.root) {
+
+    fun bind(item: Scoreboard) {
+        binding.avatarImage.setImageResource(R.drawable.icon1) //TODO Player-Avatar
+        binding.tvPlayerName.text = item.playerName
+        binding.tvScore.text = item.score.toString()
```

Comment:
This is probably confusing for the player, how do they know that the number is the score? - (because right now only a number will show - ex. "249") - i would expand this using buildString (for usage look GameModelDebugger.kt line 8) and make something like: "Score: XXX" or "Score XXX" etc. etc.